index.html

```diff
@@ -0,0 +1,213 @@
+              <img src="./src/images/photos/category-headphones.jpg" class="pictures__image">
+            </div>
+          </div>
+          <p class="category__name">Headphones & earbuds</p>
+        </article>
+
+        <article class="category section__category">
+          <div class="pictures">
+            <div class="pictures__item pictures__item--big">
+              <img src="./src/images/photos/category-speakers-01.jpg" class="pictures__image">
+            </div>
+
+            <div class="pictures__item pictures__item pictures__item--small">
+              <img src="./src/images/photos/category-speakers-02.jpg" class="pictures__image">
+            </div>
+          </div>
+          <p class="category__name">Speakers</p>
+        </article>
+
+        <article class="category section__category">
+          <div class="pictures">
+            <div class="pictures__item pictures__item--small">
+              <img src="./src/images/photos/category-audio-sunglasses-01.jpg" class="pictures__image">
+            </div>
+
+            <div class="pictures__item pictures__item--big">
+              <img src="./src/images/photos/category-audio-sunglasses-02.jpg" class="pictures__image">
+            </div>
+          </div>
+          <p class="category__name">Audio sunglasses</p>
+        </article>
+
+        <a href="#" class="button section__button">View all</a>
+      </section>
+
+      <section class="section main__section" id="howtobuy">
+        <div class="about">
+          <h2 class="title title--secondary about__title">
+            Why buy direct from Bose
+          </h2>
+          <div class="benefits about__benefits">
+            <p class="benefits__highlight">
+              Free 2-day shipping and returns
+            </p>
+
+            <p class="benefits__highlight">
+              90-day risk-free trial
+            </p>
+
+            <p class="benefits__highlight">
+              World class customer service
+            </p>
+
+            <p class="benefits__highlight">
+              My Bose account management
+            </p>
+
+            <p class="benefits__paragraph">
+              A great product is more than what’s in the box. It’s a promise of premium performance, world-class
+              support,
+              and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with
+              confidence on Bose.com.
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="section main__section" id="contacts">
+        <h2 class="title title--secondary main__title">
+          Contact us
+        </h2>
+        <div class="contact">
+          <form class="form contact__form">
+            <div class="form__item">
+              <label for="name" class="form__label">Name</label>
+              <input id="name" class="form__input" type="text" placeholder="Name" required>
```

Comment:
All inputs HTML tags should also contain `name` attribute to make sure data are added to the payload after submitting form. Please add it to all inputs in the project, it should point the same name as included in label `for` attribute e.g. 
```suggestion
              <input id="name" class="form__input" name="name" type="text" placeholder="Name" required>
```